src/+FeatureCreators/LCDFeatureSet.m

```diff
@@ -0,0 +1,202 @@
+classdef LCDFeatureSet < FeatureCreators.Base
+% Largest Common Denominator feature set Specifies a feature set consisting of:
+%   see LCDFeatureSet.getAFErequests()
+
+    %% --------------------------------------------------------------------
+    properties (SetAccess = private)
+        deltasLevels;
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Static)
+    end
+    
+    %% --------------------------------------------------------------------
+    methods (Access = public)
+        
+        function obj = LCDFeatureSet( )
+            obj = obj@FeatureCreators.Base();
+        end
+        %% ----------------------------------------------------------------
+
+        function afeRequests = getAFErequests( obj )
+            commonParams = {...
+                'pp_bNormalizeRMS', true, ...
+                'pp_intTimeSecRMS', 4, ...
+                'pp_bBinauralRMS', false, ...
+                'fb_type', 'gammatone', ...
+                'fb_lowFreqHz', 80, ...
+                'fb_highFreqHz', 8000, ...
+                'ihc_method', 'halfwave', ... % stream segr. uses 'dau'
+                'ild_wSizeSec', 20E-3, ...  % DnnLoc uses 20E-3, stream segr. uses 25E-3
+                'ild_hSizeSec', 10E-3, ...
+                'rm_wSizeSec', 20E-3, ... % DnnLoc uses 20E-3, identification uses 25E-3
+                'rm_hSizeSec', 10E-3, ... % DO NOT CHANGE -- important for gabor filters
+                'rm_scaling', 'power', ... % DnnLoc uses power, identification uses magnitude

```

Comment:
I actually believe magnitude vs power will not make a very big impact on identification, but has to be tested. @ningma97 , what's your motivation of using power vs magnitude?